paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveCatalogITCaseBase.java

```diff
@@ -251,12 +251,18 @@ public void testCreateExternalTable() throws Exception {
 
     @Test
     public void testFlinkWriteAndHiveRead() throws Exception {
```

Comment:
I just run this test, it is OK without your fixing.

Reply:
When I verified for the first time, I could reproduce the problem submitted by the user, but now when I verify again, the query data is normal, and I have begun to doubt my life.